konnectivity-client/pkg/client/client.go

```diff
@@ -339,7 +339,15 @@ func (t *grpcTunnel) serve(tunnelCtx context.Context) {
 			conn, ok := t.conns.get(resp.ConnectID)
 
 			if !ok {
-				klog.V(1).InfoS("Connection not recognized", "connectionID", resp.ConnectID)
+				klog.ErrorS(nil, "Connection not recognized", "connectionID", resp.ConnectID)
+				t.stream.Send(&client.Packet{
```

Comment:
ideally we would add ObservePacket() and conditional ObserveStreamError() here.

(See the other 2 stream send in this file, and note that t.stream is the raw stream.)

Reply:
Done. We may want to consider switching to a wrapper around the ProxyStream client that records the metrics automatically.